src/aat/java/uk/gov/hmcts/ccd/datastore/tests/helper/idam/OAuth2.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.ccd.datastore.tests.helper.idam;
+
+import uk.gov.hmcts.ccd.datastore.tests.Env;
+
+public enum OAuth2 {
```

Comment:
this seems to me a class more than an enum

Reply:
question: how come we're not using Spring? In that case you could simply have this as a singleton bean. 
Also I imagine this functional tests 'sub project' is going to grow in the future so all spring goodness might be useful

Reply:
Because Spring is slow to start, and because those tests are run in 5 steps of the pipeline, every added second is multiplied by 5. I find it hard to justify the use of Spring when our needs are a few singletons and a bit of DI.

Reply:
👍